### PR TITLE
add finnish translation

### PR DESCRIPTION
**Pull request by oh1jw**
finnish translation

### CODE REVIEW

**Comment by gxgpet** [comment]
Hello!

Is this work-in-progress? I see some untranslated lines.

**Comment by oh1jw** [comment]
There's no good translation for some of the technical terms. Of course it's possible to translate those, but that makes no sense.

**Comment by gxgpet** [comment]
I see. Well, while I'm not an expert in Finnish, I still find it hard to believe that the Finnish language doesn't have linguistic anchors for literally any modern technical term or phrase. But, as this is an open-source repository of translations, any post-contributions are welcomed as well, if we find further improvements possible.

Don't forget to submit your credits in the README file. And let me know when you think I could merge this PR.

File: language/finnish/calendar_lang.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * System messages translation for CodeIgniter(tm)
+ *
+ * @author CodeIgniter community
+ * @copyright Copyright (c) 2014-2019, British Columbia Institute of Technology (https://bcit.ca/)
```
Comment (gxgpet) [inline language/finnish/calendar_lang.php]: Since these files are new, please use the appropriate year for the copyright notice.

**Comment by oh1jw** [comment]
> I see. Well, while I'm not an expert in Finnish, I still find it hard to believe that the Finnish language doesn't have linguistic anchors for literally any modern technical term or phrase. But, as this is an open-source repository of translations, any post-contributions are welcomed as well, if we find further improvements possible.

Well - believe me when I tell you, that here the most technical related terms are still used in english. 
But, it would not be the first time when I'm wrong. I hope that in the future other Finnish speakers will take part in this translation process.
> 
> Don't forget to submit your credits in the README file. And let me know when you think I could merge this PR.

I will fix the years, submit credits and let you know.

File: language/finnish/calendar_lang.php (hunk 1 above)
Comment (oh1jw) [reply language/finnish/calendar_lang.php]: fixed

**Comment by oh1jw** [comment]
ready for merging

**Comment by gxgpet** [comment]
Thank you for your contribution!

**Pull request merged**